ex9/js/filter.js

```diff
@@ -0,0 +1,40 @@
+define(function() {
+  return function(filter) {
+    $('#btnMax').click(function() {
+      console.log('searchOn');
+      $( '#btnFilter' ).prop( "disabled", true );
+      //$('#filter').val('Type your filter');
+      $('.searchMin').hide();
+      $('.searchMax').show();
+    });
+
+    $('#btnMin').click(function() {
+      console.log('searchOff');
```

Comment:
Clicking on this should undo the filter. Currently it only affects the filter UI.